DumpCementeryDB.sql

```diff
@@ -19,106 +19,106 @@
 DB Structure Goes Here...
 */
 
-/* ####### Create Sector ####### */
+/* ####### Create Sectors ####### */
```

Comment:
Use singular for table names (user, role), and not plural (users, roles). The plural could lead to some weird table names later (instead of user_has_role, you would have users_have_roles, etc.)

Reply:
https://pl.wikipedia.org/wiki/Encja_(bazy_danych)
Read this. I was guided by this article